config/initializers/rack_attack.rb

```diff
@@ -5,6 +5,11 @@ def remote_ip
       @remote_ip ||= (env["action_dispatch.remote_ip"] || ip).to_s
     end
   end
+
+  BLOCKED_IPS = ENV.fetch("RACK_ATTACK_BLOCKED_IPS", "").split(",").map(&:strip).freeze # Array of IPs to block
```

Comment:
I have extracted it to a constant so it computes the blocked IP list once when starting the app and doesn't do the computation again for each request.